Formula/f/flang.rb

```diff
@@ -62,20 +62,6 @@ def install
     system "cmake", "--build", "build"
     system "cmake", "--install", "build"
 
-    libexec.install bin.children
-    bin.install_symlink libexec.children
-
-    # Our LLVM is built with exception-handling, which requires linkage with the C++ standard library.
-    # TODO: Remove this if/when we've rebuilt LLVM with `LLVM_ENABLE_EH=OFF`.
-    flang_cfg_file = if OS.mac?
-      resource_dir = Utils.safe_popen_read(llvm.opt_bin/"clang", "-print-resource-dir").chomp
-      resource_dir.gsub!(llvm.prefix.realpath, llvm.opt_prefix)
-      ["-lc++", "-Wl,-lto_library,#{llvm.opt_lib}/libLTO.dylib", "-resource-dir=#{resource_dir}"]
```

Comment:
Accidentally removed too much here. Will need to restore resource dir to fix ARM macOS (https://github.com/Homebrew/homebrew-core/issues/205639)